src/Microsoft.PowerShell.ConsoleGuiTools/ConsoleGui.cs

```diff
@@ -67,10 +100,19 @@ public HashSet<int> Start(ApplicationData applicationData)
             return selectedIndexes;
         }
 
+        private void Accept(){
+            Application.RequestStop();
+        }
+
+        private void Close(){
+            _cancelled = true;
+            Application.RequestStop();
+        }
+
         private Window AddTopLevelWindow()
         {
             // Creates the top-level window to show
-            var win = new Window(_applicationData.Title)
+            var win = new ConsoleGuiWindow(_applicationData.Title ?? "Out-ConsoleGridView")
```

Comment:
I believe the `OutConsoleGridViewCommand` class already sets the default to `"Out-ConsoleGridView"` so I don't think you need to do that here.